packages/iov-bns/src/bnsconnection.spec.ts

```diff
@@ -363,6 +404,109 @@ describe("BnsConnection", () => {
     connection.disconnect();
   });
 
+  it("can add address to username and remove again", async () => {
```

Comment:
Can you test removing before adding, to show the failure when it is not registered (or just removing some random non-matching value)?

This test can be enhanced much more when we support queries to view current state, until then, mainly the check of what does and doesn't work.

Reply:
ok: https://github.com/iov-one/iov-core/pull/535/commits/f18008d6521d1c052f91659dcb76991a637df5e6